manifests/v1beta1/components/new-ui/deployment.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
```

Comment:
Please can you check the latest manifest for the UI: https://github.com/kubeflow/katib/tree/master/manifests/v1beta1/components/ui.
We've made few changes to to the labels and `imagePullPolicy`